configuration-cr.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: harbor-configuration.harbor.configuration/v1alpha1
+kind: HarborConfiguration
```

Comment:
Why is this one in the project root and not in `samples`?

Reply:
this is from the initial POC sample yaml file which we are not using anymore, I'll remove it